Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCAudioModule.h

```diff
@@ -155,7 +156,7 @@ class LibWebRTCAudioModule : public webrtc::AudioDeviceModule {
 
 #if PLATFORM(COCOA)
     uint64_t m_currentAudioSampleCount { 0 };
-    bool m_isRenderingIncomingAudio { false };
+    uint64_t m_isRenderingIncomingAudioCounter { 0 };
```

Comment:
can't this be accessed on multiple thread? if not, shouldn't this be atomic?

otherwise, can we add a threading assertion?

Reply:
It is main thread only

Reply:
I'll add a main thread assertion.